### ISSUE TITLE
Goroutine state filter

### ISSUE THREAD

**Issue by kapitanov** [kapitanov/goscope.net#117]
We should have a filter on goroutines - a drop-down list of items.
Items should be displayed based on the data - if there is a goroutine in "running" state, then we should have "running" in the drop-drop filter.

Filter should have few quick selection buttons:

- "select all"
- "select none"
- "running only" (checks "running" and "runnable" items if available)
- "all but running" (checks "running" and "runnable" items if available)

Filter should be applied only after the drop-down is closed. An "Apply" button would be nice to have.

![image](https://github.com/kapitanov/goscope.net/assets/1442286/aa968126-2ca0-40ec-b934-5564fb58f046)
